models/groups_users.js

```diff
@@ -1,113 +1,113 @@
 const db = require("../data/db-config");
 
 module.exports = {
-	add,
-	find,
-	update,
-	remove
+  add,
+  find,
+  update,
+  remove
 };
 
 function add(group_user) {
-	return (
-		db("groups_users")
-			// user_id, user_type, group_id required
-			.insert(group_user, ["*"])
-			.then(g_u => find({ "g_u.id": g_u[0].id }).first())
```

Comment:
It actually needs to be this filter key in quotes for the aliased db reference to recognize the tablename -- same down below for the `update` function's nested .find call